rviz_default_plugins/src/rviz_default_plugins/displays/pointcloud/point_cloud_common.cpp

```diff
@@ -259,12 +259,8 @@ void PointCloudCommon::updateStyle()
 void PointCloudCommon::updateBillboardSize()
 {
   auto mode = static_cast<rviz_rendering::PointCloud::RenderMode>(style_property_->getOptionInt());
-  float size;
-  if (mode == rviz_rendering::PointCloud::RM_POINTS) {
-    size = point_pixel_size_property_->getFloat();
-  } else {
-    size = point_world_size_property_->getFloat();
-  }
+  float size = getSizeForRenderMode(mode);
```

Comment:
Can probably be `const`.
```suggestion
  const float size = getSizeForRenderMode(mode);
```